templates/default/my-ndb.cnf.erb

```diff
@@ -14,6 +14,8 @@ bind-address    = <%= @my_ip %>
 pid-file        = <%= node['ndb']['log_dir'] %>/mysql_<%= @mysql_id %>.pid
 tmpdir          = /tmp
 
+# Explicit defaults for TS needed by 'airflow initdb'
+explicit_defaults_for_timestamp = 1
```

Comment:
This file is not overwritten in case of an update.

Reply:
Changed resource action to ":create"